src/sql/workbench/browser/modelComponents/infoBox.component.ts

```diff
@@ -72,6 +72,7 @@ export default class InfoBoxComponent extends ComponentBase<azdata.InfoBoxCompon
 			this._infoBox.infoBoxStyle = this.style;
 			this._infoBox.text = this.text;
 			this._infoBox.isClickable = this.isClickable;
+			this._infoBox.ariaLabel = this.ariaLabel;
```

Comment:
this is not the aria-label for the info box, it is for the button inside the info box, you should introduce new properties.